sources/packages/backend/libs/sims-db/src/entities/application.model.ts

```diff
@@ -383,6 +383,10 @@ export interface ApplicationData {
    * Offering id selected by the student.
    */
   selectedOffering?: number;
+  /**
+   * Offering start date of the selected offering.
+   */
+  selectedOfferingDate?: string;
```

Comment:
It can be removed based on the response for the question https://github.com/bcgov/SIMS/pull/2902/files#r1506504337